### ISSUE TITLE
Broken on Lineage OS 21 (Android 14)

### ISSUE THREAD

**Issue by jay-ramani** [nongthaihoang/custom_font_installer#12]
Did everything the documentation says, and yet get a "/sbin/mirror/system/etc/fonts.xml not found" error while installing via Magisk.

Version: custom_font_installer-20211008
Device: Lenovo Tablet TB-X606F
ROM: GSI at https://sourceforge.net/projects/andyyan-gsi/files/lineage-21-td/lineage-21.0-20241118-UNOFFICIAL-arm64_bgN-signed.img.gz/download

**Comment by nongthaihoang** [comment]
Your version is too old. Download the one from the download button on the README.

**Comment by jay-ramani** [comment]
There is no download button on the README file. No download-able links either. Could you please share the link you are referring to?

I had previously picked up the latest release on your release page
![image](https://github.com/user-attachments/assets/740f76d9-e05d-4bb0-974b-5f9a52ca0122)

**Comment by nongthaihoang** [comment]
![Screenshot_20241223-122852_GitHub.png](https://github.com/user-attachments/assets/b75ff7b0-7882-4f04-903a-6330d6415b86)

Here

**Comment by jay-ramani** [comment]
It worked like a charm with your link on LineageOS! Thank you. However, it doesn't on Android 15 based ROMs like [crDroid](https://xdaforums.com/t/rom-15-ksu-beryllium-crdroid-v11-0-17-12-2024.4704307/) and [EvolutionX](https://xdaforums.com/t/rom-15-beryllium-official-evolution-x-10-1-2024-12-29.4684021/) though, even when set to the default font setting with these ROMs (since they have their own limited set of fonts to choose from). Would you have any suggestions on how we can get it working on these?

Thanks and cheers.

**Comment by nongthaihoang** [comment]
Crdroid 11 here. Working fine. For best results, choose the sans-serif from the font option. Works on LOS 22 as well. Tested by users.

**Comment by jay-ramani** [comment]
I setup the configuration file (attached, renamed to a text file to allow GitHub to upload) to use the [Fira Sans Extra Condensed font](https://fonts.google.com/specimen/Fira+Sans+Extra+Condensed), placed the configuration file and the downloaded font zip file from Google Fonts under the under the ```OhMyFont``` directory in internal storage. I made sure the ```OhMyFont/CFI``` directory was empty before flashing, as per the instructions.

While flashing the [module](https://github.com/nongthaihoang/custom_font_installer/raw/master/release/CFI.zip) with the above details worked on a [LineageOS 18 based ROM](https://xdaforums.com/t/bah2-guide-install-a11-from-emui-8-fix-audio.4520305/) on my Huawei MediaPad M5 Lite tablet, it doesn't work on Android 15 based [crDrdoid 11](https://xdaforums.com/t/rom-15-ksu-beryllium-crdroid-v11-0-17-12-2024.4704307/) and [EvolutionX](https://xdaforums.com/t/rom-15-beryllium-official-evolution-x-10-1-2024-12-29.4684021/) ROMs for the Xiaomi Poco F1 phone, with setting the font to Sans Serif in the font options as well. The flashing itself succeeds, but the font in question is not taking effect after rebooting with these Android 15 ROMs. What am I doing wrong? Would you be able to share your configuration file set to use the font above from Google Fonts?

Thanks again for patiently helping.
[config.cfg](https://github.com/user-attachments/files/18391722/config.cfg.txt)

**Comment by nongthaihoang** [comment]
You must read the instructions again. 
This is the OMF folder and config should look like:
![Screenshot_20250113-074707_MiXplorer.png](https://github.com/user-attachments/assets/c3159a83-269b-44fb-afb5-575a295c59fd)

![Screenshot_20250113-074829_MiXplorer.png](https://github.com/user-attachments/assets/f0fe8689-052c-4c05-ac7b-da632eeec672)

**Comment by jay-ramani** [comment]
Yes, that's what my OMF directory looks like as well.

I found out where the mistake was - in the configuration file, I had not used underscores in between the words as in the file name! Once I fed in the underscores, I got it working. I'm so glad! Hope this thread saves anyone else making the mistake I did.

Thanks much for the assistance.

Just a suggestion though - since you said the release version is outdated, how about making a release from the master branch to reflect the latest code?
